benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -65,6 +67,22 @@ class BenchmarkBase(ABC):
     # number of iterations used to run when collecting instruction_count or compile_time_instruction_count.
     _num_iterations = 5
 
+    def __init__(
+        self,
+        model_type: str,
+        device: str,
+        backend: str = "",
+        mode: str = "",
+        dynamic=False,
+        fullgraph=False,
+    ):
+        self._model_type = model_type
+        self._device = device
+        self._backend = backend
+        self._mode = mode  # Training or inference
+        self._dynamic = dynamic
+        self._fullgraph = fullgraph
```

Comment:
I do not think we need the _fullgraph? we do not have benchmarks that run with and without fullgraph modes.
i also do not think it would be interesting to filter on this